task-manager-backend/src/middlewares/is-admin-middleware.js

```diff
@@ -0,0 +1,13 @@
+const ApiError = require("../utils/api-error");
+
+const { UNAUTHORIZED_REQUEST } = require("../commons/constants");
+
+const isAdminMiddleware = (req, _, next) => {
+    if (req.session?.isAdmin) {
```

Comment:
lets always avoid else whereever possible 
this can be something like below (early exit)
if (req.session?.isAdmin) {
        return next();
    }
    throw new ApiError(401, UNAUTHORIZED_REQUEST);

Reply:
Okay